caps/capabilities.go

```diff
@@ -92,8 +97,14 @@ func (r *Repository) Add(cap *Capability) error {
 
 // Remove removes the capability with the provided name.
 // Removing a capability that doesn't exist silently does nothing
-func (r *Repository) Remove(name string) {
-	delete(r.caps, name)
+func (r *Repository) Remove(name string) error {
```

Comment:
Given that the build-in delete is not returning an error if the element is not there, I wonder if its important for us. Note that I don't really mind, I'm just curious about the why?